maraboupy/MarabouUtils.py

```diff
@@ -47,3 +48,16 @@ def addAddend(self, c, x):
         """
         self.addendList += [(c, x)]
 
+    def isEqualTo(self, equation):
+        """
+        Add a comparison between two equations.
+
+        :param equation: the other equation to be compared with.
+        :return: True if these two equations are identical; False if not.
+        """
+        equivalence = True
```

Comment:
Could just do instead of introducing a variable equivalence

return self.addendList != equation.addendList \
                or self.scalar != equation.scalar \
                or self.EquationType != equation.EquationType